docs/content/developer/getting-started/local-network.mdx

```diff
@@ -273,6 +272,3 @@ pnpm add <PATH_TO_YOUR_REPO>/sdk/typescript
 ```
 
 and the locally compiled version of `@iota/iota-sdk` package will be installed for your application.
-
-
-<Quiz questions={questions} />
```

Comment:
Will we have a quizz later or the page just doesn't deserves one?

Reply:
Added a quiz ;)